src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -43,7 +43,7 @@ export default class StripeUPEPaymentStrategy implements PaymentStrategy {
 
         const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(`${gatewayId}?method=${methodId}`));
         const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
-        const { initializationData: { stripePublishableKey, stripeConnectedAccount } } = paymentMethod;
+        const { initializationData: { stripePublishableKey, stripeConnectedAccount, shopperLanguage } } = paymentMethod;
```

Comment:
Do we need to get locale as part of paymentMethod? Or do we need to respect the locale shopper has selected as part of checkout process?

Reply:
This variable respects the language the merchant selected. If the merchant selects `Enable automatic translation based on shopper’s browser language` then this variable will have the shopper browser language.